tests/oeedger8r/edl/pointer.edl

```diff
@@ -685,7 +718,9 @@ enclave {
             [in, count=uint64_t_count] int* b16,            
             [in, count=wchar_t_count] int* b17,            
             [in, count=long_long_count] int* b18,            
-            [in, count=long_double_count] int* b19,   
+            [in, count=unsigned_long_count] int* b19,            
```

Comment:
@vishnumohan1991 - Please remove changes for unsigned long in your PR as I am addressing this with my PR #1120 .  Also do a manual merge of unsigned long long after my PR is merged to master. Thanks.